src/test/resources/application.properties

```diff
@@ -10,8 +10,8 @@ quarkus.datasource.jdbc.url=jdbc:postgresql://192.168.1.139:5432/notifications-t
 # Flyway minimal config properties
 quarkus.flyway.migrate-at-start=true
 
-#quarkus.http.access-log.enabled=true
-quarkus.http.access-log.category=info
+quarkus.http.access-log.enabled=true
```

Comment:
I disabled this one on purpose, it simply spams the test log for no good reason.